README.md

```diff
@@ -8,37 +8,39 @@
 
 <p align="center"><img src="https://raw.githubusercontent.com/hackclub/meta/5243af92814b6daacadd66e1342ad073e023544c/logos/hackedu_letter_opaque.png" alt="Hack Club"/></p>
 <p align="center">
-<b><a href="workshops/README.md">workshops</a></b>
-|
-<b><a href="case_studies/">case studies</a></b>
-|
-<b><a href="meta/labs/">labs</a></b>
+  <a href="https://hackclub.io)">Hack Club</a> is the movement of student-led coding clubs.
 </p>
 
 -------------------------------------------------------------------------------
 
-[Hack Club](https://hackclub.io) is the movement of student-led coding clubs.
-Want to start a Hack Club at your school? Apply over at
-https://hackclub.io/apply and [join our Slack](https://slack.hackclub.io) in the
-meantime. Regardless of whether you're part of Hack Club, we encourage you to
-check out and contribute to this repository.
+## Where to Start:
 
-Important links:
+| Action                                  | Link                                        |
+|-----------------------------------------|---------------------------------------------|
+| Chat with us _(we're here almost 24/7)_ | https://slack.hackclub.io                   |
+| Start a club                            | https://hackclub.io/apply                   |
+| See our workshops                       | http://workshops.hackclub.io                |
 
-- [Workshops](workshops/README.md) :: Workshops for use in clubs
-- https://slack.hackclub.io :: Join our [Slack](https://slack.com/) community
+## About
 
-## Questions? Comments? Concerns?
+Hack Club works with high school students to start programming. We provide all
+the tools for starting and leading clubs including a baseline coding curriculum
+and a community of other club leaders.
```

Comment:
Let's change "Hack Club works with high school students to start programming." to something targeted to the user. Maybe "Hack Clubs are after-school communities at high schools where people build things together. This repo contains the tools for starting and leading great clubs.